README.md

```diff
@@ -1,47 +1,47 @@
-# completable-futures [![Build Status](https://travis-ci.org/spotify/completable-futures.svg?branch=master)](https://travis-ci.org/spotify/completable-futures) [![Maven Central](https://img.shields.io/maven-central/v/com.spotify/completable-futures.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.spotify%22%20AND%20a%3A%22completable-futures%22) [![License](https://img.shields.io/github/license/spotify/completable-futures.svg)](LICENSE)
+# completable-futures
+[![Build Status](https://travis-ci.org/spotify/completable-futures.svg?branch=master)](https://travis-ci.org/spotify/completable-futures)
+[![Test Coverage](https://codecov.io/github/spotify/completable-futures/coverage.svg?branch=master)](https://codecov.io/github/spotify/completable-futures?branch=master)
+[![Maven Central](https://img.shields.io/maven-central/v/com.spotify/completable-futures.svg)](https://maven-badges.herokuapp.com/maven-central/com.spotify/completable-futures])
+[![License](https://img.shields.io/github/license/spotify/completable-futures.svg)](LICENSE)
 
-completable-futures is a set of small utility functions to simplify working with asynchronous code
-in Java8.
+completable-futures is a set of utility functions to simplify working with asynchronous code in
+Java8.
 
-This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code.
+## Usage
 
-[code-of-conduct]: https://github.com/spotify/code-of-conduct/blob/master/code-of-conduct.md
-
-#### Build dependencies
-* Java 8 or higher
-* Maven
-
-#### Runtime dependencies
-* Java 8 or higher
-
-### Import
-
-completable-futures is meant to be used as a library in other software. To import it with maven,
-use this:
+Using `completable-futures` requires Java 8 but has no additional dependencies. It is meant to be
+included as a library in other software. To import it with maven, add this to your pom:
 
 ```xml
 <dependency>
     <groupId>com.spotify</groupId>
     <artifactId>completable-futures</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
+    <version>0.1.0</version>
 </dependency>
 ```
 
+## Features
+
 ### Combining more than two things
 
+The builtin CopmletableFuture API includes `future.thenCombine(otherFuture, function)` but if you
+want to combine more than two things it gets trickier. The `CompletableFutures` class contains the
+following APIs to simplify this use-case:
+
 #### allAsList
```

Comment:
I tried that but struggled to find concise headings without writing a full sentence. Still think this is better than what we had before though.